nixos/modules/programs/gphoto2.nix

```diff
@@ -11,12 +14,20 @@ with lib;
       enable = mkOption {
         default = false;
         type = types.bool;
+        description = "Whether to configure system to use gphoto2.";
+      };
+      group = lib.mkOption {
         description = ''
-          Whether to configure system to use gphoto2.
-          To grant digital camera access to a user, the user must
-          be part of the camera group:
-          `users.users.alice.extraGroups = ["camera"];`
+          Group created when gphoto2 is enabled.
+
+          To grant digital camera access to a user, the user must be part of this group:
+          `users.users.alice.extraGroups = [config.programs.gphoto2.group];`.
+
+          By default, a group named `${defaultGroup}` will be created.
```

Comment:
```suggestion
```
the default value of the option is already documented in all docs.

Also then we can drop the let in entirely.